src/js/components/projects/detail.tsx

```diff
@@ -75,10 +63,61 @@ const ProjectDetail = (props: RouteComponentProps) => {
         }
       });
     }
-  };
+  }, [dispatch, epics?.next, isMounted, project?.id]);
+
   // create modal related
-  const openCreateModal = () => setCreateModalOpen(true);
-  const closeCreateModal = () => setCreateModalOpen(false);
+  const openCreateModal = useCallback(() => setCreateModalOpen(true), []);
+  const closeCreateModal = useCallback(() => setCreateModalOpen(false), []);
+
+  // guided tour related
+  const closeTourLandingModal = useCallback(
+    () => setTourLandingModalOpen(false),
+    [],
+  );
+  const doRunTour = useCallback(
+    (type: TourType) => {
+      if (user.onboarded_at) {
+        setTourRunning(type);
```

Comment:
@jgerigmeyer these are the code branches I mentioned in standup that aren't covered in the tests; could be why the tests for header are not passing as well.